schema.sql

```diff
@@ -4,5 +4,6 @@ CREATE TABLE animals(
 	date_of_birth DATETIME,
 	escape_attempts INT,
 	neutered BOOLEAN,
-	weight_kg DECIMAL
+	weight_kg DECIMAL,
+	species VARCHAR(50)
```

Comment:
- [ ] @mikemtzp great work adding the column, but since you had already created the table you are supposed to use `ALTER` to add column species to your table.

Reply:
Hello @danmainah!

Thanks for the observation! That's exactly what I did but mistakenly added it inside of the `CREATE TABLE`.
![image](https://user-images.githubusercontent.com/97407840/183750472-a2431064-5d84-4d58-aee0-5ac1c7183b43.png)

I have already modified the file with the correct command. 😁